pkg/cmd/safe/commands.go

```diff
@@ -22,11 +24,30 @@ func isVerbSafe(verb string) (bool, error) {
 	if verb == "" {
 		return true, nil
 	}
-	commands, err := getSafeCommands()
+
+	// first check if the verb is configured as unsafe
+	unsafeCommands, err := getUnSafeCommands()
+	if err != nil {
+		return false, err
+	}
+	if len(unsafeCommands.cmds) > 0 {
+		// if unsafeCommands contains the verb, it is unsafe
+		return !unsafeCommands.Contains(verb), nil
+	}
+	// second check if it configured as safe
+	safeCommands, err := getSafeCommands()
 	if err != nil {
 		return false, err
 	}
-	return commands.Contains(verb), nil
+	return safeCommands.Contains(verb), nil
+}
+
+func getUnSafeCommands() (*Commands, error) {
```

Comment:
nit: does this function really add value?

you can use `unsafeCommands, err := parseCommands(KubectlUnsafeCommands)` instead of `unsafeCommands, err:= getUnSafeCommands()` and nothing will break IIUC

Reply:
i agree. i just like to be explicit but maybe this is being a little too verbose